sources/models/src/aws-dev/defaults.d/31-send-metrics-aws.toml

```diff
@@ -0,0 +1 @@
+../../../shared-defaults/send-metrics-aws.toml
```

Comment:
nit. Do we want the filename to be this specific? If we decide to regionalize other metrics aspects we would need to create yet another toml.

Reply:
I think for now I'll leave the filename, since other metrics metadata is currently shared, but we can pretty easily change it if we do an expansion.